### PR TITLE
On-Prem System Requirements

### PR DESCRIPTION
**Pull request by resmaa**
Sub Page added to on-Premises Installation 
  System Requirements
    1. Quick Start Version
    2. Enterprise Version

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/355#" title="Author self-approved">resmaa</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/355#" title="Author self-approved">resmaa</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:66]: Can add Screenshots?

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:64]: Same with table is listed below. So might be remove this?

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:175]: No. DNS Mapping is providing a proper URL by which Spectro Cloud Console can be accessed. 
Can be mentioned something like:
An URL mapped to internal/external DNS server which can be used to access Spectro Cloud Console

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:177]: Also for Password resets and Alert notifications too.

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:178]: Nope.

Provide SSL certificate to access Spectro Cloud Console in a secured https way.

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:189]: Can mention minimum requirement. And also we can mention that the installer will take care of allocating the resources.

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:200]: No Quick Start Version

can be 
while installing Spectro Cloud On-Prem Quick Start Application

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:37]: Will not be required here. As it differs for QS and enterprise.

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:196]: The instances should be reachable to the whitelisted domains either through Proxy or Without Proxy Network Connections.

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:208]: Mention minimum required

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:216]: The instances should be reachable to the whitelisted domains either through Proxy or Without Proxy Network Connections.

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:217]: In Bracket or as a note can mention the tagging part

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:226]: As this On prem we can mention:

A Private Cloud Gateway is a Spectro Cloud component that enables the communication between Spectro Cloud's management console and a data center which is not accessible directly via the Datacenter hosting Spectro Cloud On-Prem Application. The gateway needs to be installed by the users in their VMware environments using a private cloud gateway installer OVA. For more details <link to Gateway installer section>

File: packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md
Comment (arvind-sys) [inline packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md:237]: esxi

**Review by jgautam — APPROVED** [review]
Looks Good. Need to review in the docs-latest site for better readability , so approving it.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/355#pullrequestreview-658131132" title="Approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/355#" title="Author self-approved">resmaa</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
